tests/backbeat/Using.md

```diff
@@ -9,18 +9,29 @@
    127.0.0.1 zenko.local
    ```
 
-2. Create an account using Orbit.
-3. Export the access key and secret key of that account (for example, in
+2. Find the Zenko service endpoint for S3 calls to Cloudserver by running:
+    ```
+    kubectl get services
+    ```
+    Look for the service named `zenko-cloudserver`, the value under `CLUSTER-IP`
+    should be set as the Zenko endpoint for testing:
+
+```
+export CLOUDSERVER_ENDPOINT=<cluster-ip of zenko-cloudserver>
```

Comment:
Thinking out loud here 💭. Maybe to follow the pattern of avoiding spaces we could use `<zenko-cloudserver-cluster-ip>`.

Reply:
updated and squashed!